shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -360,5 +381,69 @@ public void dispatchSemanticsAction(
     public int systemGestureInsetRight = 0;
     public int systemGestureInsetBottom = 0;
     public int systemGestureInsetLeft = 0;
+    public List<DisplayFeature> displayFeatures = List.of();
+  }
+
+  /**
+   * Area of the viewport obstructed by a feature, eg. a hinge, a fold crease or camera cutout
+   *
+   * <p>Similar to {@link androidx.window.DisplayFeature}, used for reporting Fold / Hinge areas.
+   * The reason we define our own model is that we also want to support cutouts.
+   */
+  public static final class DisplayFeature {
+    public final Rect bounds;
```

Comment:
It is called `bounds` here. We should keep this naming consistent in the framework/dart:ui/embedders to avoid future confusion.